typescript/test/deposit-sweep.test.ts

```diff
@@ -686,25 +688,7 @@ describe("Sweep", () => {
               // Validate inputs.
               expect(txJSON.inputs.length).to.be.equal(2)
 
-              const p2wshInput = txJSON.inputs[0]
-              expect(p2wshInput.prevout.hash).to.be.equal(
-                depositSweepWithNonWitnessMainUtxoAndWitnessOutput.deposits[0].utxo.transactionHash.toString()
-              )
-              expect(p2wshInput.prevout.index).to.be.equal(
-                depositSweepWithNonWitnessMainUtxoAndWitnessOutput.deposits[0]
-                  .utxo.outputIndex
-              )
-              // Transaction should be signed. As it's a SegWit input, the `witness`
-              // field should be filled, while the `script` field should be empty.
-              expect(p2wshInput.witness.length).to.be.greaterThan(0)
-              expect(p2wshInput.script.length).to.be.equal(0)
-              // Input's address should be set to the address generated from deposit
-              // script hash
-              expect(p2wshInput.address).to.be.equal(
-                "tb1qk8urugnf08wfle6wslmdxq7mkz9z0gw8e6gkvspn7dx87tfpfntshdm7qr"
-              )
-
-              const p2pkhInput = txJSON.inputs[1] // main UTXO
```

Comment:
Why do we need to move this?

Reply:
See https://github.com/keep-network/tbtc-v2/pull/700#discussion_r1338583359.